### PR TITLE
HNMI non-root Docker image

### PR DESCRIPTION
**Pull request by andyzito**
Goes with https://github.com/greenriver/hnmi/pull/1192

### CODE REVIEW

File: lib/roll_out.rb [hunk 1]
```diff
@@ -214,14 +214,14 @@ def deploy_web!
...
-      container_port: 443,
+      container_port: 3000,
     }]
 
     minimum, maximum = _get_min_max_from_desired(web_options['container_count'])
 
     _start_service!(
       capacity_provider: _capacity_provider_name,
-      name: name,
+      name: name.gsub(/v2/,'v3'),
```
Comment (yakloinsteak) [inline lib/roll_out.rb]: Is this just for testing?

**Comment by andyzito** [comment]
@yakloinsteak Not sure why GitHub isn't letting me reply in the thread, but in answer to your question re: line 224 (.gsub):
- I think it's preferable to use version bumping rather than delete the service and recreate. It means no downtime, and is generally safer.
  - we DO have to create/recreate the service, b/c you can't change the load balancing settings (where the port is configured) on an existing Service
- However, I agree that the way I did it here is not ideal, and I was wondering if you had any thoughts on how we want to handle versioning in this case. We could bump the whole thing to hnmi-v3, but that would mean rebuilding the entire infrastructure, which seems unnecessary

**Comment by yakloinsteak** [comment]
What I was thinking about is that I originally wanted this to be a generic repo to be used in open path and new deployments (but I've never gotten far on that). I'd hope for this to be something we could pass in, or at least make this change only apply to hnmi (to make refactoring later easier). Just search for "hnmi" in this repo, and you'll see what I mean.

**Comment by andyzito** [comment]
@yakloinsteak what do you think about the change in the most recent commit? HNMI would pass in:
```
  versions: {
    :ecs_service => '2'
  }
```
And then that would get appended to the service name. It wouldn't append anything if `versions` didn't contain a matching key. Could be extended to version as many individual bits as we need.

**Comment by yakloinsteak** [comment]
Sounds like a good plan.

**Review by yakloinsteak — APPROVED** [review]
Go for it, assuming you've had a successful deployment with it.

**Pull request merged**